virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -75,7 +75,7 @@ The SVVP Certification applies to:
 //CNV-28096 Release note: NEW feature -- clone from cached snapshot
 * You can now enable volume snapshots as boot sources.
 
-//CNV-28724 Release note: storage profile defaults
+//CNV-28724 Release note: Storage profile defaults
 * The access mode and volume mode fields in storage profiles are populated automatically with their optimal values for the following additional Containerized Storage Interface (CSI) provisioners:
 
 ** Dell PowerFlex
```

Comment:
Can't comment on lines below -- I recommend that you remove the abbreviations in parentheses because they are unnecessary. This is the only time we mention the various provisioners in this module.